ocaml/testsuite/tests/typing-layouts-err-msg/value.ml

```diff
@@ -2,7 +2,6 @@
    flags = "-extension layouts_alpha"
    * expect
 *)
-(* CR layouts v2.9: all error messages below here are not reviewed for quality *)
```

Comment:
The error message beginning on line 37 reads:
```
Error: Variables bound in a class must have layout value.
       The layout of v is float64, because
         it's the type of a variable bound by a `let`, defaulted to layout float64.
       But the layout of v must be a sublayout of value, because
         it's let-bound in a class expression.
```
The reason given for `v` says that the type of a variable was "defaulted to layout float64", but this doesn't make sense - when we default, we default to value.

Reply:
Oops, I see there's a CR just below on that point.  But perhaps until we can address that CR we should pick some other word than "defaulted" for whatever is happening here.

Reply:
This is changed in `main` to throw a different type of error. The incorrect part about `defaulted to layout float64` should disappear when rebased